From 3bdd7e40e1c7e7484edcb53b01ef3cd446305466 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Mon, 15 Nov 2010 15:07:49 +0300
Subject: [PATCH 08/12] Fix path to CGI executables

See rhbz #653291 for additional details:

https://bugzilla.redhat.com/653291

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>


Modified for 4.0.8 - Scott Wilkerson <swilkerson@nagios.com>

---
 sample-config/httpd.conf.in | 4 ++--
 subst.in                    | 1 +
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/sample-config/httpd.conf.in b/sample-config/httpd.conf.in
index 71d4fbb..1b1e693 100644
--- a/sample-config/httpd.conf.in
+++ b/sample-config/httpd.conf.in
@@ -5,9 +5,9 @@
 # configuration file.  Customize the paths, etc. as
 # needed to fit your system.
 
-ScriptAlias @cgiurl@ "@sbindir@"
+ScriptAlias @cgiurl@ "@cgidir@"
 
-<Directory "@sbindir@">
+<Directory "@cgidir@">
 #  SSLRequireSSL
    Options ExecCGI
    AllowOverride None
diff --git a/subst.in b/subst.in
index a8e2c06..bbffb3a 100755
--- a/subst.in
+++ b/subst.in
@@ -35,6 +35,7 @@ while ($f = shift @ARGV) {
 		s|\@datarootdir\@|@datarootdir@|g;
 		s|\@sbindir\@|@sbindir@|g;
 		s|\@bindir\@|@bindir@|g;
+		s|\@cgidir\@|@libdir@/cgi-bin/|g;
 		s|\@htmurl\@|@htmurl@|g;
 		s|\@cgiurl\@|@cgiurl@|g;
 		s|\@MAIL_PROG\@|@MAIL_PROG@|g;
-- 
1.8.3.1

